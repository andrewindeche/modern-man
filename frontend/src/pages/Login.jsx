import React from 'react';
const Login = () => {
    return(
        <div className="loginForm">
             <form>
            <p>Login: Log into your Account</p>
                <span className="nameLabel">
                    <div className="input-group">
                        <label>Email or Phone</label>
                        <input type="text" name="fullName" placeholder="Enter your Email or Phone" required />
                    </div>
                </span>
                <span className="contactLabel">
                    <div className="input-group">
                        <label>Password</label>
                        <input type="text" name="email" placeholder="Enter your Password" required />
                    </div>
                </span>
                <span className="contactLabel">
                    <div className="input-group">
                        <label>Forgot Password?<a className="signupnow">Reset Password</a></label>
                    </div>
                    <div className="input-group">
                        <label>Not a member?<a className="signupnow">Sign Up Now</a></label>
                    </div>
                </span>
                    <button type="submit">Login</button>
            </form>
        </div>
    )
}
export default Login;