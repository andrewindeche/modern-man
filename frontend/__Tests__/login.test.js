import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Login from 'pages/Login.jsx';
describe('Forgot component', () => {
  it('renders without crashing', () => {
    const { getByText } = render(
      <Router>
        <Login />
      </Router>
    );
    expect(getByText(/Log into your Account?/i)).toBeInTheDocument();
  });
});
