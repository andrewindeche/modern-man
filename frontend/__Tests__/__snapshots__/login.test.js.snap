// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login page renders the login page 1`] = `
<div
  className="loginForm"
>
  <form>
    <p>
      Login: Log into your Account
    </p>
    <span
      className="nameLabel"
    >
      <div
        className="input-group"
      >
        <label>
          Email or Phone
        </label>
        <input
          name="fullName"
          placeholder="Enter your Email or Phone"
          required={true}
          type="text"
        />
      </div>
    </span>
    <span
      className="contactLabel"
    >
      <div
        className="input-group"
      >
        <label>
          Password
        </label>
        <input
          name="email"
          placeholder="Enter your Password"
          required={true}
          type="text"
        />
      </div>
    </span>
    <span
      className="contactLabel"
    >
      <div
        className="input-group"
      >
        <label>
          Forgot Password?
          <div
            className="signupnow"
          >
            Reset Password
          </div>
        </label>
      </div>
      <div
        className="input-group"
      >
        <label>
          Not a member?
          <div
            className="signupnow"
          >
            Sign Up Now
          </div>
        </label>
      </div>
    </span>
    <button
      type="submit"
    >
      Login
    </button>
  </form>
</div>
`;
